src/scroll_list/ScrollList.js

```diff
@@ -379,6 +380,17 @@ define(function(require) {
          */
         this.onScrollToItemStarted = Observable.newObservable();
 
+        /**
+         * Observable for subscribing to scroll events beyond AwesomeMap boundaries.
+         *
+         * @method AwesomeMap#onScrollPastTopBoundary
+         * @param {Function} callback
+         *        Invoked with (sender, {
+         *            boundary: {@link BoundaryTypes}
+         *        })
+         */
+        this.onScrollPastBoundary = Observable.newObservable();
```

Comment:
I was thinking:  It might be better to have an observable for each direction, rather than the types in a generic observable.  Then consumers could subscribe to the boundaries they care about, without having to check which boundary was reached on each firing of the observable.  

It means more code in the library, but I think there would be less repeated code checking for the boundary that was reached.  Then again, they might have to subscribe to more observables, if they care about all boundaries.  Maybe you have more information about the expected use case that would clarify how the API should be.